gems/pending/spec/util/postgres_admin_spec.rb

```diff
@@ -55,5 +55,23 @@
           .to eql "su - postgres -c '/ctl/path stop -W -D /pgdata/path -s -m fast'"
       end
     end
+
+    describe ".initialized?" do
+      around do |example|
+        Dir.mktmpdir do |dir|
+          ENV["APPLIANCE_PG_DATA"] = dir
```

Comment:
Are we capturing the value before and restoring it after?  I don't know if I like mocking `ENV` here but we do something similar in terms of backup/restore the value [here](https://github.com/ManageIQ/manageiq/blob/1f1786b5a2577581d350a8cb40c57d45b40fe8f3/spec/lib/extensions/database_configuration_spec.rb#L26-L35)